src/node/player.js

```diff
@@ -0,0 +1,47 @@
+const prompt = require('prompt-sync')();
+const constants = require('./constants.js');
+const Point = require('./point.js');
+
+function Player() {
+  const player = {
+    isWinner: false,
+    symbol: ' ',
+    move(board) {
+      let userMove = player.promptMove();
+      while (inputIsInvalid(userMove, board)) {
+        userMove = player.promptMove();
+      }
+      return userMove;
+    },
+    promptMove() {
+      console.log(`Vez de ${player.symbol}. Qual sua jogada? (linha, coluna)`);
+      const userInput = prompt();
+      return Point(Number(userInput[0]), Number(userInput[3]));
+    },
+  };
+  return player;
+}
+
+function inputIsInvalid(userMove, board) {
+  if (Number.isNaN(userMove.x) || Number.isNaN(userMove.y)) {
+    console.log('Movimento invalido!');
```

Comment:
Always have the user who will use your game in mind.
If you already have the conditions to give more information about the error to the user, why wouldn't you?
Here, for example: `Movimento invalido! Você digitou ${userMove.x || userMove.y}, porém apenas numeros são aceitos` would be a better message to the user. And it's only possible, because your `if` conditional was well made and filtered the errors to be only `not a number` case! Same with the other errors, all of them know which error occurred, so why improve your user experience, even when he fails to do something?